.github/workflows/lint-and-test.yml

```diff
@@ -63,7 +63,7 @@ jobs:
           cache: 'poetry'
       - name: Set up MySQL
         run: |
-          sudo systemctl start mysql
+          sudo systemctl start mysqld
```

Comment:
I couldn't find much info about whether to use `mysql` or `mysqld`, but I know the current behavior works for the OS that we test.

Do we need to add a check here like:
```
if systemctl list-units --type=service | grep -i mysqld > /dev/null; then
    sudo systemctl start mysqld
else
    sudo systemctl start mysql
fi
```

Reply:
I further verified. Your current code with `mysql` is right. I close this PR.